### PR TITLE
[*] Composite Primary Keys - Forest handle null values.

### PR DESCRIPTION
**Pull request by lucasscariot**
## Pull Request checklist:

- [x] Write an explicit title for the Pull Request
- [x] Write changes made in the CHANGELOG.md
- [ ] Create automatic tests
- [x] Test manually the implemented changes
- [x] Review my own code (indentation, syntax, style, simplicity, readability)
- [x] Wonder if you can improve the existing code

### CODE REVIEW

**Comment by lucasscariot** [comment]
[(?) Sequelize Composite Primary keys should work if one the CPK fields is left empty.](https://trello.com/c/GGYaAKNo/1735-sequelize-composite-primary-keys-should-work-if-one-the-cpk-fields-is-left-empty)

File: services/composite-keys-manager.js [hunk 1]
```diff
@@ -8,6 +8,10 @@ function CompositeKeysManager(model, schema, record) {
     var where = {};
     var primaryKeyValues = recordId.split(GLUE);
 
+    primaryKeyValues.forEach(function (key, index) {
```
Comment (arnaudbesnier) [inline services/composite-keys-manager.js]: Can you add a comment to explain why we have to do that here (something that could happen with a bad model configuration)?

File: services/composite-keys-manager.js [hunk 2]
```diff
@@ -20,6 +24,9 @@ function CompositeKeysManager(model, schema, record) {
     var compositePrimary = '';
 
     _.keys(model.primaryKeys).forEach(function (primaryKey, index) {
+      if (record[primaryKey] === null) {
```
Comment (arnaudbesnier) [inline services/composite-keys-manager.js]: Same comment here?

**Comment by arnaudbesnier** [comment]
Fix: https://github.com/ForestAdmin/forest-express-sequelize/issues/124

**Pull request merged**